src/SFML/Audio/SoundFileReaderFlac.cpp

```diff
@@ -75,10 +76,9 @@ FLAC__StreamDecoderTellStatus streamTell(const FLAC__StreamDecoder*, FLAC__uint6
 {
     auto* data = static_cast<sf::priv::SoundFileReaderFlac::ClientData*>(clientData);
 
-    const std::int64_t position = data->stream->tell();
-    if (position >= 0)
+    if (const auto position = data->stream->tell())
```

Comment:
Ditto
```suggestion
    if (const auto position = data->stream->tell(); position.has_value())
```

Reply:
Same here, I'm not trying to be needlessly pedantic but I can imagine reading this code and thinking that we're relying on `position != 0` instead of `position` being an optional. It's also the variable name that doesn't suggest any optionality.